### DISCUSSION TITLE
Tensorflow TypeError: Unable to convert function return value to a Python type! The signature was 	() -> handle

### DISCUSSION THREAD

**Discussion by skabirgithub** [pyinstaller/pyinstaller#8001] [PyInstaller]
File "tensorflow\python\framework\dtypes.py", line 37, in <module>
TypeError: Unable to convert function return value to a Python type! The signature was
	() -> handle

I have a hook file named hook-tensorflow.py:..................
from PyInstaller.utils.hooks import collect_all
def hook(hook_api):
    packages = [
        'tensorflow',
        'tensorflow_core',
        'astor'
    ]
    for package in packages:
        datas, binaries, hiddenimports = collect_all(package)
        hook_api.add_datas(datas)
        hook_api.add_binaries(binaries)
        hook_api.add_imports(*hiddenimports)
        
        
        
        
        
        my tensorflow version is 2.14.0 and pyinstaller version is 6.0

**Comment by rokm** [comment]
See #7961 - for 2.14.0, you need to add `ml_dtypes` to hidden imports.
